lms/djangoapps/course_blocks/transformers/start_date.py

```diff
@@ -43,6 +43,26 @@ def name(cls):
         """
         return "start_date"
 
+    @classmethod
+    def _check_has_scheduled_content(cls, block_structure, scheduled_content_condition):
```

Comment:
Why pass in the method rather than just having that logic inline? This doesn't feel like a super generic method otherwise, such that you'd want that re-usability.

Reply:
I thought it would be cleaner if I pulled that code out into a separate method, but you're right it's probably overkill

Reply:
Naw I see now that you use the passed in lambda in two places... So I dig your code re-use. I might have just made another classmethod and referenced it in two places, rather than passing it in? But your use makes more sense to me now that I see it's used twice.